README.md

```diff
@@ -28,6 +28,23 @@ To install the SDK into your project, simply run:
 
     npm install nahmii-sdk
 
+## Dependency management
+
+After you have installed the nahmii-sdk module into your own project, please 
+make sure to install the dependencies listed as _peer dependencies_ into your 
+project to have it included in the end product.
```

Comment:
What will happen if someone forgets to do this? Or simply doesn't read this part of the documentation?

Reply:
It will crash saying it cant find the necessary module. Also, NPM will complain when you do an "npm install" if there are missing peer dependencies!